package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "chart.js": "^4.4.6",
+    "react-chartjs-2": "^5.2.0"
+  }
```

Comment:
what are those dependencies, they are not used in your components at all?

also, absoluletly make sure NOT to commit node_modules!!! Isn't that in your gitignore? gitignore should be commited as the team.